src/master-playlist-controller.js

```diff
@@ -1296,8 +1306,12 @@ export class MasterPlaylistController extends videojs.EventTarget {
    *
    * @param {Object=} error an optional error that may include the playlist
    * to blacklist
+   * @param {Number=} blacklistDuration an optional number of seconds to blacklist the
+   * playlist
+   * @param {Boolean=} skipPlaylistSelection an optional boolean to skip playlist
+   * selection
    */
-  blacklistCurrentPlaylist(error = {}) {
+  blacklistCurrentPlaylist(error = {}, blacklistDuration, skipPlaylistSelection = false) {
```

Comment:
Good call on the bandwidth being artificial, therefore not triggering. Will do that instead.